### DISCUSSION TITLE
How to check ble printer allready coonnected

### DISCUSSION THREAD

**Discussion by lajith111** [capacitor-community/bluetooth-le#566] [Q&A] [answered]
I want to know about  2 questions..below is to code to ble intialize.

async Initialize() {
    await BleClient.initialize({ androidNeverForLocation: true });
  }
1)Whether i have to call intialize before ble connect request every time or once at page constructor (ionic)?

2) if ble printer is already conected. I could not find isconected function in this library

Please some one suggest me

**Comment by pwespi** [comment, marked as answer]
1. The `initialize` call should only be made once for you whole application.
2. To get a list of connected devices, there is the [getConnectedDevices](https://github.com/capacitor-community/bluetooth-le#getconnecteddevices) method.
